src/tools/qt-gui/src/guisettings.cpp

```diff
@@ -41,14 +41,23 @@ void GUISettings::setDefaults ()
 	palette.setCurrentColorGroup (QPalette::Disabled);
 
 	m_useSystemIconTheme = true;
+	m_viewermode = false;
 }
 
-GUISettings::GUISettings (QObject * parentGUISettings)
-: QObject (parentGUISettings), m_profile ("/current/"), m_base ("/sw/elektra/qtgui/#0/"), m_highlightColorString ("color/highlight"),
-  m_frameColorString ("color/frame"), m_nodeWKeyColorString ("color/node/with"), m_nodeWOKeyColorString ("color/node/without"),
-  m_useSystemIconThemeString ("icon/system"), m_legacyBase ("/sw/libelektra.org/qt-gui/#0/"),
-  m_legacyHighlightColorString ("highlight_color"), m_legacyFrameColorString ("frame_color"),
-  m_legacyNodeWKeyColorString ("node_with_key_color"), m_legacyNodeWOKeyColorString ("node_without_key_color")
+GUISettings::GUISettings (QObject * parentGUISettings): QObject (parentGUISettings)
+,m_profile ("/current/")
```

Comment:
Do you have troubles installing clang-format-3.8? Otherwise please run scripts/reformat-source